src/test/groovy/au/org/ala/profile/hub/AdminControllerSpec.groovy

```diff
@@ -0,0 +1,31 @@
+package au.org.ala.profile.hub
+
+import grails.testing.web.controllers.ControllerUnitTest
+import org.apache.http.HttpStatus
+import spock.lang.Specification
+
+class AdminControllerSpec extends Specification implements ControllerUnitTest<AdminController> {
+
+    AdminController mockAdminController
+
+    def setup() {
+        mockAdminController = Mock(AdminController)
+    }
+
+    def "cache management should display to the admin controller when admin mode"() {
+        when:
+        mockAdminController.cacheManagement()
```

Comment:
No need to mock controller. Use `controller.cacheManagement`.  use [this file](https://github.com/AtlasOfLivingAustralia/profile-hub/blob/dev/src/test/groovy/au/org/ala/profile/hub/BHLControllerSpec.groovy#L55) as example.